rfd/0058-package-distribution.md

```diff
@@ -0,0 +1,66 @@
+| Who provides signing key                   | Teleport                   | Teleport                                                                                                     | Teleport                                                                | Either                                         | Google                                                                            |
+| OOTB third party secret provider support   | AWS, Hashicorp, Kubernetes | AWS, Hashicorp, Kubernetes                                                                                   | Hashicorp                                                               | No                                             | N/A                                                                               |
+| APT channel support via components         | No ('main' only)           | Yes                                                                                                          | Yes                                                                     | No ('main' only)                               | No ('main' only)                                                                  |
+| APT channel support via distribution       | No ('stable' only)         | Yes                                                                                                          | Yes                                                                     | No (Specific OS versions only)                 | Yes (via separate repositories)                                                   |
+| APT channel support via URI                | No (one URI only)          | Yes                                                                                                          | Yes                                                                     | Yes                                            | Yes                                                                               |
+| YUM channel support via URI                        | No (one URI only)          | Yes                                                                                                          | Yes                                                                     | Yes                                            | Yes                                                                               |
+| Overall channel support rating             | 1/5                        | 5/5                                                                                                          | 4/5                                                                     | 2/5                                            | 3/5                                                                               |
```

Comment:
How did you come to these ratings?

Reply:
Removed - rating was more of a non-quantitative ranking used for an internal discussion that I forgot to remove.